### ISSUE TITLE
Hi, Adam.

### ISSUE THREAD

**Issue by cabuhatprince** [adamsanderson/jQuery-File-Validator#2]
Hi, I just want to ask how can i validate the image pixel? because the data-max-size is for size image only. Thankyou :)

**Comment by adamsanderson** [comment]
No, I'm afraid it doesn't validate the dimensions of an image, just the
file type and size.
